pandas/tests/series/test_api.py

```diff
@@ -245,10 +245,11 @@ def test_tab_completion(self):
 
     def test_tab_completion_with_categorical(self):
         # test the tab completion display
-        ok_for_cat = ['categories', 'codes', 'ordered', 'set_categories',
```

Comment:
Why was this changed needed? IIUC, then name, index, and categorical should *not* be included here anymore.

Reply:
I wasn't sure either, when I was running the tests locally, this was raising an AssertionError. Can remove it and see how the tests do.

Reply:
Sorry, you will need the change you had originally, not sure what I was thinking of last night.

These are going to be in `dir(Series.cat)`, because you're adding properties for the deprecation. I think they weren't there original, because those attributes were set in the `__init__`, and IPython was analyzing the class before that was called.

Could you add a note in the property `CategoricalAccessor.categorical` saying that this test will need to be updated too when the deprecation is enforced?

Reply:
Ah no worries.

When you say note on that property, is a comment sufficient, or do you mean in the deprecation FutureWarning message?